collections/retail/general-notifications/event-general-notifications/message-received/event-general-notifications.json

```diff
@@ -45,6 +46,31 @@
           ]
         }
       ]
+    },
+    {
+      "id": "new-message-received-mailout",
+      "name": "New Message Received Mailout",
+      "userPreferenceEnabled": false,
+      "additionalFilterLogic": "#triggeredEvent['origin'] != null && #triggeredEvent['origin'] == 'engagement'",
+      "deliveryOptions": [
+        {
+          "channel": "in-app-notification",
+          "channelSettings": "/new-message-received-mailout/in-app-notification/in-app_channel-settings.json",
+          "isEnabledByDefault": true,
+          "isManageableByUser": false,
+          "routeInfo": {
+            "whereTo": "conversation-view",
+            "data": "{\"id\":\"{{event.conversationId}}\"}"
+          },
+          "engagementTemplates": [
+            {
+              "path": "/new-message-received-mailout/in-app-notification/en_US/engagement-template.json",
```

Comment:
do you need to create `in-app-notification` folder under `new-message-received-mailout`?

Reply:
thanks @afsinka-backbase I missed it during the copy/ paste. It is fixed now